osu.Game/Skinning/Editor/SkinBlueprint.cs

```diff
@@ -2,56 +2,105 @@
         /// Whether the blueprint should be shown even when the <see cref="SelectionBlueprint{T}.Item"/> is not alive.
         /// </summary>
-        protected virtual bool AlwaysShowWhenSelected => false;
+        protected virtual bool AlwaysShowWhenSelected => true;
+
+        protected override bool ShouldBeAlive => drawable.IsAlive && Item.IsPresent;
 
-        protected override bool ShouldBeAlive => (drawable.IsAlive && Item.IsPresent) || (AlwaysShowWhenSelected && State == SelectionState.Selected);
+        [Resolved]
+        private OsuColour colours { get; set; }
 
         public SkinBlueprint(ISkinnableComponent component)
             : base(component)
         {
         }
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
+        private void load()
         {
             InternalChildren = new Drawable[]
             {
                 box = new Container
                 {
-                    Colour = colours.Yellow,
                     Children = new Drawable[]
                     {
-                        new Box
+                        outlineBox = new Container
                         {
                             RelativeSizeAxes = Axes.Both,
-                            Alpha = 0.2f,
-                            AlwaysPresent = true,
+                            Masking = true,
+                            BorderThickness = 3,
+                            BorderColour = Color4.White,
+                            Children = new Drawable[]
+                            {
+                                new Box
+                                {
+                                    RelativeSizeAxes = Axes.Both,
+                                    Alpha = 0f,
+                                    AlwaysPresent = true,
+                                },
+                            }
                         },
-                    }
+                        new OsuSpriteText
+                        {
+                            Text = Item.GetType().Name,
+                            Font = OsuFont.Default.With(size: 10, weight: FontWeight.Bold),
+                            Anchor = Anchor.BottomRight,
+                            Origin = Anchor.TopRight,
+                        }
+                    },
                 },
             };
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            updateSelectedState();
+            box.FadeInFromZero(200, Easing.OutQuint);
+        }
+
+        protected override void OnSelected()
+        {
+            // base logic hides selected blueprints when not selected, but timeline doesn't do that.
+            updateSelectedState();
+        }
+
+        protected override void OnDeselected()
+        {
+            // base logic hides selected blueprints when not selected, but timeline doesn't do that.
+            updateSelectedState();
+        }
```

Comment:
comment update:
```suggestion
        protected override void OnSelected()
        {
            // base logic hides selected blueprints when not selected, but skin blueprints don't do that.
            updateSelectedState();
        }

        protected override void OnDeselected()
        {
            // base logic hides selected blueprints when not selected, but skin blueprints don't do that.
            updateSelectedState();
        }
```
or can just remove it, whichever you feel best.